### DISCUSSION TITLE
Questions before closing ticket.

### DISCUSSION THREAD

**Discussion by furia09** [discord-tickets/bot#121] [General]
*How do you rate the assistance provided?.
- Reaction:
(1-2-3-4-5).

*Were you professionally helped by the staff?
- Reaction:
(YES - NO)

``If YES selected =
new reaction:
please rate:
(1-2-3-4-5)

If NO selected = 
New field to place last comment before closing ticket.

And maybe this one as Stats for the Owner:

*Did you like this ticket system?
- Reaction:
(1-2-3-4-5).

-Something that gives the owner more sight on how members and users feel about the bot.

In case member got a new suggestion then a link to suggestions channel or something related to that.

**Comment by eartharoid** [comment]
`How do you rate the assistance provided?` and `Were you professionally helped by the staff?` sound like different ways of asking the same thing.

**Comment by furia09** [comment]
Well not really

like in my word that would be

the assistence = Like the info and the tools provided to solve the members issue

Professionally = Was it done professionally or did the support  just give the right info and tolls but he was been a asshole for a example.


Probably i didn't explain myself so well

sorry mate i'm from EU

Kind regards
________________________________
Van: Isaac <notifications@github.com>
Verzonden: zaterdag 6 maart 2021 19:58
Aan: discord-tickets/bot <bot@noreply.github.com>
CC: furia09 <r.baez@live.nl>; Author <author@noreply.github.com>
Onderwerp: Re: [discord-tickets/bot] Questions before closing ticket. (#121)


How do you rate the assistance provided? and Were you professionally helped by the staff? sound like different ways of asking the same thing.
